src/Commands/Publish/PublishUserCommand.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+namespace InfyOm\Generator\Commands\Publish;
+
+use InfyOm\Generator\Utils\FileUtil;
+
+class PublishUserCommand extends PublishBaseCommand
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'infyom.publish:user';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Publishes users files';
+
+    /**
+     * Execute the command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $this->copyView();
+        $this->updateRoutes();
+        $this->updateMenu();
+        $this->publishUserController();
+        $this->publishUserRepository();
```

Comment:
there also be check if repository_pattern skip not need to publish UserRepository